### DISCUSSION TITLE
Why the Gaussian keyword is required for reading?

### DISCUSSION THREAD

**Discussion by phys-chem** [deepmodeling/dpdata#699] [General]
My Gaussian keyword:
`#  wb97xd/6-31G*  force`

When I read Gaussian outputs, I get this error:
> dpdata/gaussian/log.py", line 70, in to_system_data
>     raise RuntimeError(
> RuntimeError: Input orientation is not found. Using Gaussian keyword `Geom=PrintInputOrient` to always print the input orientation.

Is the input orientation already available in outputs even without this keyword?

**Comment by njzjz** [comment]
By default, it is only printed when the number of atoms is less than 50.

**Comment by njzjz** [comment]
Gaussian output files also contain a copy of the user input GJF file. In theory, the program can read this section instead of `Input orientation`, though it doesn't at present. You are welcome to contribute.
